macros/wrap_string_with_quotes.sql

```diff
@@ -1,7 +1,14 @@
 {% macro wrap_string_with_quotes(str) %}
-  {% if not str %}
+  {% if str is none %}
     {{ return('cast(NULL as ' ~ dbt.type_string() ~ ')') }}
   {% else %}
-    {{ return("'" ~ str ~ "'") }}
+    {{ dbt.string_literal(str) }}
   {% endif %}
 {% endmacro %}
+
+{#
+  To be removed when https://github.com/dbt-labs/dbt-bigquery/pull/1089 is merged
```

Comment:
Let's touch base with adapters team to see if this is something we can merge asap